### PR TITLE
Feature/pn 626

### PR DESCRIPTION
**Pull request by MTurraPpa**

### CODE REVIEW

File: runtime-infra/03_API-GW.yaml [hunk 1]
```diff
@@ -381,11 +383,32 @@ Resources:
...
+      Principal: apigateway.amazonaws.com
+      SourceArn: !Sub "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${WebapiGwRestApi}/authorizers/${ApikeyAuthorizer}"
+
+  ApikeyAuthorizer:
+    Type: AWS::ApiGateway::Authorizer
+    Properties:
+      Name: apikeyAuthorizer
+      Type: REQUEST
+      AuthorizerResultTtlInSeconds: '300'
+      AuthorizerUri: !Sub 'arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-central-1:558518206506:function:apikeyAuthorizer/invocations'
```
[the remark this answers is not in the document]
Comment (MTurraPpa) [reply runtime-infra/03_API-GW.yaml:407]: Infatti: Authorizer name must be <= 1024 characters in length and must match the character set: [a-zA-Z0-9._-]+

File: runtime-infra/04_WEBAPI-GW.yaml [hunk 2]
```diff
@@ -0,0 +1,370 @@
+# Sample: https://github.com/aws-samples/aws-apigw-http-api-private--integrations/blob/main/templates/APIGW-HTTP-private-integration-ALB-ecs.yml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: WEBAPI gateway Piattaforma Notifiche
```
[the remark this answers is not in the document]
Comment (MTurraPpa) [reply runtime-infra/04_WEBAPI-GW.yaml]: fatto

File: runtime-infra/04_WEBAPI-GW.yaml [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+      Principal: apigateway.amazonaws.com
+      SourceArn: !Sub "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${WebapiGwRestApi}/authorizers/${JwtAuthorizer}"
+
+  JwtAuthorizer:
+    Type: AWS::ApiGateway::Authorizer
+    Properties:
+      Name: JwtAuthorizer
+      Type: TOKEN
+      AuthorizerResultTtlInSeconds: '300'
+      AuthorizerUri: !Sub 'arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-central-1:558518206506:function:jwtAuthorizer/invocations'
```
[the remark this answers is not in the document]
Comment (MTurraPpa) [reply runtime-infra/04_WEBAPI-GW.yaml:361]: fatto

**Review by marco-vit-pagopa — APPROVED** [review]
Ok

**Pull request merged**